services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -4215,54 +4198,38 @@ private void internalUpdateStore(String clusterName, String storeName, UpdateSto
         storeMetadataUpdate(clusterName, storeName, store -> {
           if (!isHybrid(finalHybridConfig)) {
             /**
-             * If all of the hybrid config values are negative, it indicates that the store is being set back to batch-only store.
+             * If all the hybrid config values are negative, it indicates that the store is being set back to batch-only store.
              * We cannot remove the RT topic immediately because with NR and AA, existing current version is
              * still consuming the RT topic.
              */
             store.setHybridStoreConfig(null);
-            // Disabling hybrid configs for a L/F store
-            if (!store.isIncrementalPushEnabled()) {
-              // Enable/disable native replication for batch-only stores if the cluster level config for new batch
-              // stores is on
-              store.setNativeReplicationEnabled(clusterConfig.isNativeReplicationEnabledAsDefaultForBatchOnly());
-              store.setNativeReplicationSourceFabric(
-                  clusterConfig.getNativeReplicationSourceFabricAsDefaultForBatchOnly());
-              store.setActiveActiveReplicationEnabled(
-                  store.isActiveActiveReplicationEnabled()
-                      || clusterConfig.isActiveActiveReplicationEnabledAsDefaultForBatchOnly());
-            } else {
-              store.setNativeReplicationEnabled(clusterConfig.isNativeReplicationEnabledAsDefaultForIncremental());
-              store.setNativeReplicationSourceFabric(
-                  clusterConfig.getNativeReplicationSourceFabricAsDefaultForIncremental());
-              store.setActiveActiveReplicationEnabled(
-                  store.isActiveActiveReplicationEnabled()
-                      || clusterConfig.isActiveActiveReplicationEnabledAsDefaultForIncremental());
-            }
+            store.setIncrementalPushEnabled(false);
+            // Enable/disable native replication for batch-only stores if the cluster level config for new batch
+            // stores is on
+            store.setNativeReplicationEnabled(clusterConfig.isNativeReplicationEnabledAsDefaultForBatchOnly());
```

Comment:
Not for this PR, but we really need to clean up NR configs. They are all enabled for a year or more, it is good time get rid of all of them...

Reply:
Yeah I agree. I even made some progress on this in the past: https://github.com/sushantmane/venice/tree/delete-nr-checks but because of other high priority tasks wasn't able to finish it.